### ISSUE TITLE
Failed to reproduce USPS -> MNIST results

### ISSUE THREAD

**Issue by geonyeong-park** [Alexander-H-Liu/UFDN#8]
Hello, 

I'm a newbie for domain adaptation field, and your work is very appreciated for studying.
I tried to reproduce the UDA results (USPS->MNIST) following your appendix parameters, as following
![image](https://user-images.githubusercontent.com/46081019/64165079-47a9f580-ce7f-11e9-85ed-98a3d8df5639.png)  
  
  
Unfortunately i failed to reproduce the results, where i found it works for default SVHN->MNIST case. If you dont mind would you please let me know any reasons for the result?

p.s. default url of usps seems to be expired, so i exchanged it with https://github.com/mingyuliutw/CoGAN/blob/master/cogan_pytorch/data/uspssample/usps_28x28.pkl

**Comment by Alexander-H-Liu** [comment]
Hi @ParkGeonYeong ,

Thank you for recognizing our work, please refer to Issue #6  for more details about MNIST<->USPS (e.g.  `data_augment` should be set to `False`).
The guy who opened the issue is able to reproduce results of both MNIST->USPS and USPS->MNIST.

Also, I'm not familiar with the new USPS you mentioned, so I can't tell if it's identical to the one I've used.

Let me know if you have further questions.

**Comment by geonyeong-park** [comment]
Thank you for further information @Alexander-H-Liu , 
There're couple of minor questions

1. Actually i added a last 'fc' layer with output dim=64 for vae encoder, since i understood your supplementary table in a way that input for generator should be 64+2  
![image](https://user-images.githubusercontent.com/46081019/64227211-96a66800-cf1d-11e9-8b23-1b0cf5ec64e6.png)
  
However, i checked out issue #6  that 'fc' layer may be not necessary for reproducing peformance.
Am i got something misunderstood with your Table? Also, don't know which part i'm missing but i hardly found the setting of discriminator for UDA on supplementary tables. Is it fine with using default .yaml?
   
  
2. Actually i'm translating your good codes into tensorflow,(planning to make it as opensource) and having some troubles with it. I'm testing with MNIST -> MNISTM with your UDA setting, and my VAE returns almost same output regardless of input. When i tried with some smaller network version with 3 Conv and fcs, at least the reconstructed images looks fine. I'm sorry to ask some ambiguous question, but is there any tips/tricks to debug such problems?

Thank you in advance.

**Comment by Alexander-H-Liu** [comment]
Hi @ParkGeonYeong,

On your first question, the `2` in `64+2` stands for the domain code mentioned in the paper. That is, the hard categorical representation which was pre-defined and given, not extracted by the encoder.

As for your second question, you should try to address the problem above and see if it gets better. I not familiar with TensorFlow, yet in my own experience, the optimal learning rate might be slightly different when switching from one to the other. (Again, it's an empirical investigation, not necessarily true)

p.s. as mentioned before and shown in issue #6 , I believe the reproducibility of my own code is not the problem now, so I'm closing the issue (you can still contact me through this issue), thanks.
